spring-cloud-commons/src/main/java/org/springframework/cloud/client/serviceregistry/Registration.java

```diff
@@ -0,0 +1,7 @@
+package org.springframework.cloud.client.serviceregistry;
+
+/**
+ * @author Spencer Gibb
+ */
+public interface Registration {
```

Comment:
I ended up putting this back in, because I needed to lookup the registration in the spring context without knowing the implementation.